src/20240118_StkGHO_Activation/StkGHOActivation.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "GHO Safety Module Initialization"
+author: "Aave_BGDLabs_ACI"
+discussions: "https://governance.aave.com/t/arfc-upgrade-safety-module-with-stkgho/15635"
+---
+
+## Simple Summary
+
+This AIP activates the new GHO based Safety module by initiating the emission schedule approved by the community during the vote 
+https://snapshot.org/#/aave.eth/proposal/0x4bc99a842adab6cdd8c7d5c7a787ee4c0056be554fde0d008d53b45b3e795065
+
+## Motivation
+
+The GHO Safety Module will fortify the Aave Protocol’s resilience by adding a stablecoin asset, which is inherently less volatile than AAVE. This strategic move diversifies the Safety Module’s capacity to absorb shocks from various risk vectors in case of shortfall events.
+
+## Specification
+
+The GHO Safety module will be activated with the following parameters:
+
+Base emission: 50 AAVE/day
+Duration: Three months
+
+## References
```

Comment:
Here links should be included. Some default ones should have been auto-generated though.
And probably good to add stkGHO proxy additionally and link to https://github.com/bgd-labs/stake-token (the implementation)